src/schemas/21-526EZ/schema.js

```diff
@@ -15,139 +14,118 @@ const directDepositDef = {
       type: 'string',
       pattern: '^\\d{4,17}$'
     },
-    routingNumber: {
-      type: 'string',
-      pattern: '^\\d{9}$'
-    },
     bankName: {
       type: 'string',
-      maxLength: 35, // Is this going to be enough?
+      maxLength: 35,
       pattern: "([a-zA-Z0-9\-'.,# ])+$"
     }
   }
-}
+};
 
-// Copy pasta from the common address definition
-const countries = constants.countries.map(object => object.value);
-const countriesWithStateList = Object.keys(constants.states).filter(x => _.includes(countries, x));
-const countryStateProperties = _.map(constants.states, (value, key) => ({
-  properties: {
-    country: {
-      type: 'string',
-      'enum': [key]
-    },
-    state: {
-      type: 'string',
-      // TODO: state is only a two-character code
-      'enum': value.map(x => x.value)
-    }
-  }
-}));
-countryStateProperties.push({
-  properties: {
-    country: {
-      not: {
+const disabilitiesBaseDef = {
+  type: 'array',
+  maxItems: 100,
+  items: {
+    type: 'object',
+    properties: {
```

Comment:
some of these properties are marked as optional, does that mean the rest are required? 
Including:
- diagnosticText
- disabilityActionType
- decisionCode
- specialIssues
- ratedDisabilityId

Reply:
I see, required is below, would it be possible to move it above?

Reply:
Also, do any additional properties need to be added to the required array?

Reply:
Looks like we're missing `disabilityActionType` based on the Swagger:
```
Disability {
diagnosticText (string)stringMax. Length:255Reg. Exp.:([a-zA-Z0-9\-'.,#]([a-zA-Z0-9\-',.# ])?)+$,
disabilityActionType (string) = ['NONE', 'NEW', 'SECONDARY', 'INCREASE', 'REOPEN']stringEnum:"NONE", "NEW", "SECONDARY", "INCREASE", "REOPEN",
decisionCode (string),
specialIssues (Array[SpecialIssue], optional)objectMax. Items:100,
ratedDisabilityId (string),
ratingDecisionId (string, optional),
diagnosticCode (integer, optional),
specialIssueTypeCode (string, optional),
secondaryDisabilities(Array[SecondaryDisability], optional)objectMax. Items:100
}
```

Reply:
Oh, good point. I'm _guessing_ it'll always be `INCREASE` for this form, but it's worth checking. Also, we should add that in either the front or back end if it's always going to be the same.

@goldenmeanie As the resident SME(ish), do you have any insight on if the `disabilityActionType` will always be `INCREASE` for this form?

Reply:
It sounds like in the future this form will support other types?

Reply:
Yes, this is just increase but All Claims will incorporate New and Secondary. I'm not sure about Reopen (but signs point to yes).